lib/riax.ex

```diff
@@ -38,6 +53,29 @@ defmodule Riax do
   def ping(key) do
     sync_command(key, {:ping, key})
   end
+  ## ------------------------ File Cache CSV -----------------
+
+  @path "/Users/fran/Downloads/covid19.csv"
```

Comment:
can this come from config? also it would be better to have the file in the project instead of a global dir